news/_posts/2021-02-19-first-public-oss-demo.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Announcing the public demonstrations for the project"
+subtitle: "We held a first session of Public open source demonstrations for the project"
+date:   2021-02-19 16:30:00
+authors: [ MathieuFerment ]
+icon: icon-video-camera
+tags: [contribute, events, community]
+---
+
+Last month, the maintainers team held a special event for the open source project: a public demonstrations session. To explain better the nature of this event, some context is needed.
+
+## Demonstrations at PrestaShop
+
+Until 2019, the maintainers team of the PrestaShop open source project was only composed of PrestaShop company employees, developers assigned by the company to work on the project. This team used many forms of organizations over years, the most recent being a mix of [SCRUM](https://www.scrum.org/) rituals and [Kanban](https://en.wikipedia.org/wiki/Kanban_(development)) method. The SCRUM rituals being followed by the team have a [demonstration](https://manifesto.co.uk/scrum-practice-sprint-demo/) ritual.
+
+The _demonstration_ in SCRUM is an event where, at the end of a _sprint_, what has been built is demonstrated to every team members. This helps everybody, inside and outside the developers team, to witness the outcome, what has been delivered. For years, this event happened regularly but only PrestaShop company employees could attend.
+
+In 2020, the split between the PrestaShop project and the PrestaShop company has grown wider. One of the major milestone of his evolution was the [welcome of external maintainers](https://build.prestashop.com/news/coreweekly-43-2020/) [@kpodemski](https://github.com/kpodemski) and [@pululuk](https://github.com/pululuk) who are not PrestaShop employees.
+
+Now that the maintainers team of PrestaShop has members who are not employees, the processes of the team must become more open too. That is why we wanted to turn this company-only event into a public event for the project!
+
+Also we hope that this event can become a meeting place for the maintainers and the community of PrestaShop, where ideas, feedbacks and smiles can be exchanged.
+
+## The first Public open source demonstrations
+
+So last month on the 28th of January, we held the first online session of what we call, for now, "Public PrestaShop open source demonstrations".
+
+This event was open for maintainers but not for community as this was our very first attempt to this sort of meeting. As we were not confident in the result, we wanted to keep restrict the audience to avoid disappointment. This first session aimed to give the concept a shot before going fully public. We have already identified multiple areas for improvement to make it better.
```

Comment:
```suggestion
Since this was our very first attempt to this type of meeting, in English, and including people from outside the PrestaShop company, this first session was open for maintainers but not for community. The objective was to give the concept a shot before going fully public. We have already identified multiple areas for improvement, but we were globally satisfied with the result.
```